tuf-spec.md

```diff
@@ -976,47 +980,37 @@ as is described for the <a>root.json</a> file.
     <a>TARGETPATH</a>.  The application may use this information to guide
     download decisions.
 
-<dfn>DELEGATIONS</dfn> is an object whose format is the following:
+  : <dfn>"keys"</dfn>
+  ::
+    A list of the public keys to verify signatures of delegated targets roles.
+    Revocation and replacement of delegated targets roles keys is done by
+    changing the keys in this field in the delegating role's metadata.
+
+  : <dfn>"delegations"</dfn>
+  ::
+    A list of <dfn>DELEGATION</dfn> objects whose format is the following:
 
 <pre highlight="json">
 {
-  "keys" : {
-      <a for="role">KEYID</a> : <a>KEY</a>,
-      ...
-  },
-  "roles" : [
-    {
-      "name": <a>ROLENAME</a>,
-      "keyids" : [ <a for="role">KEYID</a>, ... ] ,
-      "threshold" : <a>THRESHOLD</a>,
-      (<a>"path_hash_prefixes"</a> : [ <a>HEX_DIGEST</a>, ... ] |
-      <a for="delegation-role">"paths"</a> : [ <a>PATHPATTERN</a>, ... ]),
-      "terminating": <a>TERMINATING</a>,
-    },
-    ...
-  ]
+  "name": <a>DELEGATION_NAME</a>,
+  (<a>"path_hash_prefixes"</a> : [ <a>HEX_DIGEST</a>, ... ] |
+  <a for="delegation-role">"paths"</a> : [ <a>PATHPATTERN</a>, ... ]),
+  "terminating": <a>TERMINATING</a>,
+  "min_roles_in_agreement": <a>NUM_ROLES</a>,
+  "roles" : [{
+      "rolename": <a>ROLENAME</a>,
+      "keyids" : [ <a for="root">KEYID</a>, ... ] ,
+      "threshold" : <a>THRESHOLD</a>
+    }, ... ]
 }
 </pre>
 
-  <a for="root">KEYID</a> and <a>KEY</a> are the same as is described for the
-  <a>root.json</a> file.
+  <a for="root">KEYID</a> and <a>THRESHOLD</a> are the same as is described
+  for the <a>root.json</a> file.
 
-  : <dfn>ROLENAME</dfn>
+  : <dfn>DELEGATION_NAME</dfn>
   ::
-    A string giving the name of the delegated role.  For example, "projects".
-
-  : <dfn>TERMINATING</dfn>
-  ::
-    A boolean indicating whether subsequent delegations should be considered.
-
-    As explained in the [Diplomat paper
-    ](https://theupdateframework.io/papers/protect-community-repositories-nsdi2016.pdf),
-    terminating delegations instruct the client not to consider future trust
-    statements that match the delegation's pattern, which stops the delegation
-    processing once this delegation (and its descendants) have been processed.
-    A terminating delegation for a package causes any further statements about a
-    package that are not made by the delegated party or its descendants to be
-    ignored.
+    A string giving the name of the delegation. For example, "projects-delegation".
```

Comment:
Nit pick: I had removed the example DELEGATION_NAME in the old draft branch. See [old draft diff](https://github.com/theupdateframework/specification/compare/fd85a8a...draft#diff-b0020aab85257910a5a31f248c201a81cb35e1939545c4b45515e9049e78cdffR883). Must have slipped back in during rebase?

Also, if it os referring to the delegation name in the metadata snippet below, that one is spelled without the s. project-delegation.

I guess both is fine.

Reply:
Great catch, I'll remove.